scene/main/node.cpp

```diff
@@ -2147,30 +2147,40 @@ int Node::get_persistent_group_count() const {
 	return count;
 }
 
-void Node::_print_tree_pretty(const String &prefix, const bool last) {
+void Node::print_tree_pretty() {
+	print_line(_get_tree_string_pretty("", true));
+}
+
+void Node::print_tree() {
+	print_line(_get_tree_string(this));
+}
+
+String Node::_get_tree_string_pretty(const String &prefix, const bool last) {
 	String new_prefix = last ? String::utf8(" ┖╴") : String::utf8(" ┠╴");
-	print_line(prefix + new_prefix + String(get_name()));
-	_update_children_cache();
```

Comment:
This line also has to be kept

Reply:
what other functions?

Reply:
`_get_tree_string` in this case

Reply:
line 2153 only or both 2153 and 2152?

Reply:
Only the second, you have to update the children cache

Reply:
done, thanks

Reply:
no sorry I forgot something

Reply:
> no sorry I forgot something

it depens
should I replace children by children_cache in all the 2 functions or only on these 2 lines?

Reply:
Look at the original code

Reply:
I think it's done
(actually compiling local)